src/apps/answers/service.py

```diff
@@ -340,6 +368,10 @@ async def _create_answer(self, applet_answer: AppletAnswerCreate) -> AnswerSchem
             answer.version,
             applet_answer.alerts,
         )
+
+        if relation and relation != Relation.admin:
+            await self._delete_temp_take_now_relation_if_exists(respondent_subject, target_subject, source_subject)
```

Comment:
I think we can get rid of this condition. The relation returned from `_get_answer_relation` is unrelated to this
```suggestion
        await self._delete_temp_take_now_relation_if_exists(respondent_subject, target_subject, source_subject)
```